model_analyzer/record/metrics_manager.py

```diff
@@ -783,13 +783,23 @@ def _print_run_config_info(self, run_config):
         for model_run_config in run_config.model_run_configs():
             perf_config = model_run_config.perf_config()
             if perf_config["request-rate-range"]:
-                logger.info(
-                    f"Profiling {model_run_config.model_variant_name()}: client batch size={perf_config['batch-size']}, request-rate-range={perf_config['request-rate-range']}"
-                )
+                if perf_config["batch-size"] != 1:
```

Comment:
I changed this because most of the time the client batch size is 1, and it's not relevant to my debug